src/azure-cli/azure/cli/command_modules/network/tests/latest/test_network_commands.py

```diff
@@ -2040,7 +2040,7 @@ def test_network_express_route(self, resource_group):
         with self.assertRaisesRegexp(CLIError, 'Please provide a complete resource ID'):
             self.cmd('network express-route gateway connection show --ids /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myrg/providers/Microsoft.Network/expressRouteGateways/aaa')
 
-    @record_only()
+    @unittest.skip('Test is wrong, please fix. rg not found')
```

Comment:
Can you submit issues to track tests to be fixed?

Reply:
Good idea

Reply:
I opened one, https://github.com/Azure/azure-cli/issues/17094